myning/objects/research_facility.py

```diff
@@ -112,15 +112,16 @@ def points_per_hour(self):
         ticks_per_hour = 60 / self.minutes_per_tick
         return ticks_per_hour * self.points_per_researcher * len(self._researchers)
 
-    def check_in(self):
+    def check_in(self, bonus: int = 1):
```

Comment:
can we make this required to avoid not passing it in when we're supposed to?

Reply:
or, what if the research facility just fetched the boost from the macguffin without having it be passed in?

Reply:
I'll make it required. I don't want the singletons referencing each other.